include/ebpf_structs.h

```diff
@@ -321,6 +321,20 @@ enum bpf_attach_type
      */
     BPF_CGROUP_INET6_RECV_ACCEPT,
 
+    /** @brief Attach type for handling IPv4 TCP connect() or UDP send
```

Comment:
we shouldn't append these to the end of the enum set to avoid renumbering existing enums?